### DISCUSSION TITLE
Alternate variant urls?

### DISCUSSION THREAD

**Discussion by charles-hill-work** [apollographql/apollo-studio-community#151] [Q&A] [answered]
Is it possible to use alternate url's with a Variant in Studio without overwriting the default?  We are running multiple gateways with managed federation.  The gateways all use the same Variant, but have different (security, throttling, etc.).  We don't wan tto add additional Variants for this, as that adds unnecessary complexity (and may not work since the gateways use the same subgraph services).  This scenario doesn't work in Apollo Sandbox because we don't have introspection enabled in our deployed gateways.

Example url's:
```
api.example.com
partners-api.example.com
bulk-scripting.example.com
```

**Comment by cheapsteak** [comment, marked as answer]
Sorry that's not currently a planned feature

Would be curious to see if this is a common pain-point to help us triage and prioritize 

In the meantime, I wonder if it might be an option to have Studio point to an endpoint that routes requests based on a header?
